examples/src/test/scala/docs/jdbc/JdbcProjectionDocExample.scala

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.jdbc
+
+import java.time.Instant
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+
+import akka.Done
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.japi.function
+import akka.persistence.jdbc.query.scaladsl.JdbcReadJournal
+import akka.projection.ProjectionContext
+
+//#handler-imports
+import akka.projection.jdbc.scaladsl.JdbcHandler
+
+//#handler-imports
+
+//#projection-imports
+import akka.projection.ProjectionId
+import akka.projection.jdbc.scaladsl.JdbcProjection
+
+//#projection-imports
+
+import akka.projection.eventsourced.EventEnvelope
+import akka.projection.eventsourced.scaladsl.EventSourcedProvider
+
+//#jdbc-session-imports
+import java.sql.Connection
+import java.sql.DriverManager
+import akka.projection.jdbc.JdbcSession
+
+//#jdbc-session-imports
+
+import akka.stream.scaladsl.FlowWithContext
+import docs.eventsourced.ShoppingCart
+import org.slf4j.LoggerFactory
+
+object JdbcProjectionDocExample {
+  // #todo
+  // TDOO
+  // #todo
+
+  //#repository
+  case class Order(id: String, time: Instant)
+  trait OrderRepository {
+    def save(connection: Connection, order: Order): Unit
+  }
+  //#repository
+
+  class OrderRepositoryImpl extends OrderRepository {
+    override def save(order: Order)(implicit connection: Connection): Unit = ???
+  }
+  val orderRepository = new OrderRepositoryImpl
+
+  // #jdbc-session
```

Comment:
would be good to also include the HibernateJdbcSession in tomorrow's release

Reply:
I was planning to add the Hibernate one for Java only. The goal is to have it ready by tomorrow.

Reply:
yes, Java only for Hibernate makes sense